gst/datarepo/gstdatareposink.c

```diff
@@ -670,7 +670,7 @@ __write_json (JsonObject * object, const gchar * filename)
   json_generator_set_root (generator, root);
   json_generator_set_pretty (generator, TRUE);
   ret = json_generator_to_file (generator, filename, &error);
-  if (!ret) {
+  if (!ret && error) {
```

Comment:
plz check condition (!ret || error)

Reply:
Thank you.
ret == false and error !=NULL

Reply:
Thank you. error is removed